docs/settings/reporting-settings.asciidoc

```diff
@@ -106,7 +106,7 @@ available, but there will likely be errors in the visualizations in the report.
 If capturing a report fails for any reason, {kib} will re-attempt other reporting job, as many times as this setting. Defaults to `3`.
 
 `xpack.reporting.capture.loadDelay`::
-Specify the {time-units}[amount of time] before taking a screenshot when visualizations are not evented. All visualizations that ship with {kib} are evented, so this setting should not have much effect. If you are seeing empty images instead of visualizations, try increasing this value. Defaults to `3s`.
+Specify the {time-units}[amount of time] before taking a screenshot when visualizations are not evented. All visualizations that ship with {kib} are evented, so this setting should not have much effect. If you are seeing empty images instead of visualizations, try increasing this value. Defaults to `3s`. *NOTE*: This setting exists for backwards compatibility, but is unused and therefore does not have an affect on reporting performance.
```

Comment:
We can add a statement to tell the reader which version deprecated this setting. I'm not sure exactly when it was deprecated, but we can say 8.0.0

Something like:
`deprecated:[8.0.0,This setting has no effect.] `

Reply:
Cool! TIL